modules/table/src/test/java/org/apache/ignite/internal/table/distributed/raft/snapshot/PartitionAccessImplTest.java

```diff
@@ -119,4 +128,56 @@ void testMinMaxLastAppliedTerm(@InjectConfiguration("mock.tables.foo {}") Tables
         assertEquals(15, partitionAccess.minLastAppliedTerm());
         assertEquals(20, partitionAccess.maxLastAppliedTerm());
     }
+
+    @Test
+    void testAddWrite() {
+        TestMvTableStorage mvTableStorage = new TestMvTableStorage(tablesConfig.tables().get("foo"), tablesConfig);
+
+        MvPartitionStorage mvPartitionStorage = mvTableStorage.getOrCreateMvPartition(TEST_PARTITION_ID);
+
+        TableSchemaAwareIndexStorage indexStorage = mock(TableSchemaAwareIndexStorage.class);
+
+        PartitionAccess partitionAccess = new PartitionAccessImpl(
+                new PartitionKey(UUID.randomUUID(), TEST_PARTITION_ID),
+                mvTableStorage,
+                new TestTxStateTableStorage(),
+                () -> List.of(indexStorage)
+        );
+
+        RowId rowId = new RowId(TEST_PARTITION_ID);
+        TableRow tableRow = mock(TableRow.class);
+        UUID txId = UUID.randomUUID();
+        UUID commitTableId = UUID.randomUUID();
+
+        partitionAccess.addWrite(rowId, tableRow, txId, commitTableId, TEST_PARTITION_ID);
+
+        verify(mvPartitionStorage, times(1)).addWrite(eq(rowId), eq(tableRow), eq(txId), eq(commitTableId), eq(TEST_PARTITION_ID));
```

Comment:
Just a comment: `times(1)` is redundant, if we omit it, it will have the same effect. But it can be used to highlight to the reader that you are expecting exactly 1 call (not more), this can be useful sometimes.

Reply:
Thanks for the tip =), I'll leave it as it is.